pc/user.cpp

```diff
@@ -332,15 +335,24 @@ void user::parse_get_product( uint32_t tok, uint32_t itok )
 
 void user::send_pending_upds()
 {
-  if ( pending_vec_.empty() ) {
+  uint32_t n_sent = 0;
```

Comment:
I think `to_send` might be a clearer variable name: to me, `n_sent` implies that that it's calculating the number of updates that have already been sent.